twilio-authenticator-sample/src/main/java/com/twilio/authenticatorsample/approvalrequests/RequestsFragment.java

```diff
@@ -174,17 +181,14 @@ private void initViews(View rootView) {
 
     private void initVars() {
         SampleApp sampleApp = (SampleApp) getActivity().getApplicationContext();
-        twilioAuthenticator = (sampleApp).getTwilioAuthenticator();
```

Comment:
let's keep TwilioAuthenticator here and remove it from the newInstance method. Normally this get injected via a dependency injection framework, but we use it like this to be able to mock this during test easily.